R/multivariate.R

```diff
@@ -558,3 +558,45 @@ muvr_analysis <- function(object, y = NULL, id = NULL, multi_level = FALSE, mult
 
   muvr_model
 }
+
+# PERMANOVA ----
+
+#' Perform PERMANOVA
+#'
+#' Performs permutational multivariate analysis of variance. Uses package called PERMANOVA.
+#'
+#' @param object a MetaboSet object
+#' @param group_col A factor containing the groups to compare.
+#' @param all_features should all features be included in FDR correction?
+#' @param transform Transformation to use in \code{IniTransform}. By default uses "Standardize columns".
+#' @param coef Coefficient to calculate continuous distances in \code{DistContinuous}. By default uses Pythagorean distances.
+#' @param ... other parameters to \code{\link{PERMANOVA}}

```

Comment:
I think this should be changed like this since this is a function from an external package.

```suggestion
#' @param ... other parameters to \code{\link[PERMANOVA]{PERMANOVA}}
```

I would also add links to the other PERMANOVA functions to the "See also" section.

Reply:
Do you mean all functions from package PERMANOVA? Or other R packages performing PERMANOVA?

Reply:
Oh, what I meant was that links to functions from outside notame don't work without specifying the package, i.e. roxygen2 needs to know what package the function (in this case, `PERMANOVA()`) is from so that the link works.

But actually, reading [the chapter from the newest edition of the R packages book](https://r-pkgs.org/man.html#links-and-cross-references) , it seems that the preferred roxygen2 syntax has changed and this should be

```
#' @param ... other parameters to [PERMANOVA::PERMANOVA]
```